core/conversion/evaluators/aten.cpp

```diff
@@ -314,22 +354,30 @@ auto aten_registrations TORCHTRT_UNUSED =
                  {"aten::size(Tensor self) -> (int[])", "aten::size.int(Tensor self, int dim) -> (int)"})})
         .evaluator(
             {c10::Symbol::fromQualString("aten::__getitem__"),
-             [](const torch::jit::Node* n, kwargs& args) -> c10::optional<torch::jit::IValue> {
-               auto list = args.at(n->input(0)).IValue()->to<c10::List<c10::IValue>>();
+             [](ConversionCtx* ctx, const torch::jit::Node* n, kwargs& args) -> c10::optional<torch::jit::IValue> {
+               auto list_input = args.at(n->input(0));
                auto idx = args.at(n->input(1)).unwrapToInt();
+               if (list_input.isIValue()){
```

Comment:
@peri044: Can we have test cases with such edge case scenarios to verify this?

Reply:
I'm not sure what other edge case scenarios might occur. the input can be a IValue for normal cases, or an ITensor (when there is a preceeding aten::size which emits ITensor). Do you have anything other than these in mind ?

Reply:
Added few utils which better detect IValue and new test cases which use them.